addons/test_mass_mailing/tests/test_blacklist.py

```diff
@@ -8,15 +8,83 @@
 
 class TestBLAccessRights(common.MassMailingCase):
 
+    def setUp(self):
+        super(TestBLAccessRights, self).setUp()
+        self.bl_rec = self.env['mail.blacklist'].create([
+            {'email': 'Not A Stark <john.snow@example.com>'},
```

Comment:
I believe we should also test/care for alternative forms of addresses, such as:
```
 - "Giant; \"Big\" Box" <sysservices@example.net>
 - "Joe Q. Public" <john.q.public@example.com>
```
These double-quoted versions are particularly frequent in the wild.
See [Appendix A.1.{1,2,3} of RFC 5322](https://tools.ietf.org/html/rfc5322#appendix-A.1) for more examples.

Reply:
Done !